Makefile

```diff
@@ -2,9 +2,10 @@
 RM := rm -f
 SSH_USER := core
 TERRAFORM_INSTALLER_URL := github.com/dcos/terraform-dcos
-DCOS_VERSION := 1.11
-KUBERNETES_VERSION ?= 1.10.7
-KUBERNETES_FRAMEWORK_VERSION ?= 1.2.2-1.10.7
+DCOS_VERSION := 1.12
+CUSTOM_DCOS_DOWNLOAD_PATH := https://downloads.dcos.io/dcos/testing/1.12.0-rc2/dcos_generate_config.sh
```

Comment:
@alejandroEsc this should now be changed to `stable/1.12.0`.

Reply:
Im a bit surprised here since it was said it was either or and not sure if the stable was landed already. Also the notes said we should use -rc2 at the least to test.